File: net/source/Templates/WebAPI/Devon4Net.WebAPI.Implementation/Business/SearchCriteria/QueueSearchCriteriaTo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Devon4Net.WebAPI.Implementation.Business.SearchCriteria
{
    public class QueueSearchCriteriaTo : AbstractSearchCriteriaTo
    {
        public static long serialVersionUID = 1L;
        public int? Modificationcounter { get; set; }
        public string Name { get; set; }
        public string logo { get; set; }
        public DateTime? Attentiontime { get; set; }
        public DateTime? Minattentiontime { get; set; }
        public bool? Active { get; set; }
    }
}

File: java-quarkus/src/main/java/com/devonfw/application/domain/tos/QueueSearchCriteriaTo.java
package com.devonfw.application.domain.tos;

import java.sql.Timestamp;

import com.devonfw.module.basic.common.api.query.StringSearchConfigTo;

import lombok.Getter;
import lombok.Setter;

/**
 * {@link SearchCriteriaTo} to find instances of
 * {@link com.devonfw.application.queuemanagement.common.api.Queue}s.
 */
@Getter
@Setter
public class QueueSearchCriteriaTo extends AbstractSearchCriteriaTo {

  private static final long serialVersionUID = 1L;

  private String name;

  private String logo;

  private String currentNumber;

  private Timestamp attentionTime;

  private Timestamp minAttentionTime;

  private Boolean active;

  private StringSearchConfigTo nameOption;

  private StringSearchConfigTo logoOption;

  private StringSearchConfigTo currentNumberOption;
}
